### PR TITLE
Update intellisense package version to RC2

### PR DESCRIPTION
**Pull request by safern**
cc: @carlossanlop

### CODE REVIEW

**Review by carlossanlop — APPROVED** [review]

**Review by stephentoub — APPROVED** [review]

**Comment by safern** [comment]
We're going to hold this PR in case we need one more update for the intellisense package and merge on the 28th.

File: eng/Versions.props [hunk 1]
```diff
@@ -146,7 +146,7 @@
     <MoqVersion>4.12.0</MoqVersion>
     <FsCheckVersion>2.14.3</FsCheckVersion>
     <!-- Docs -->
-    <MicrosoftPrivateIntellisenseVersion>3.0.0-preview-20200715.1</MicrosoftPrivateIntellisenseVersion>
+    <MicrosoftPrivateIntellisenseVersion>5.0.0-preview-20200914.1</MicrosoftPrivateIntellisenseVersion>
```
Comment (carlossanlop) [inline eng/Versions.props:149]: @safern https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=164260&view=logs&j=fd490c07-0b22-5182-fac9-6d67fe1e939b&t=11e7ea89-affe-5194-cdc6-0171c3394706
```suggestion
    <MicrosoftPrivateIntellisenseVersion>5.0.0-preview-20200924.1</MicrosoftPrivateIntellisenseVersion>
```

**Pull request merged**